frontend/src/pages/org/dashboard.ts

```diff
@@ -394,6 +398,71 @@ export class Dashboard extends LiteElement {
     `;
   }
 
+  private renderCardSkeleton = () =>
+    html`
+      <sl-skeleton class="mb-3" effect="sheen"></sl-skeleton>
+      <sl-skeleton class="mb-3" effect="sheen"></sl-skeleton>
+      <sl-skeleton class="mb-3" effect="sheen"></sl-skeleton>
+      <sl-skeleton class="mb-3" effect="sheen"></sl-skeleton>
+    `;
+
+  // TODO fix style when data-table is converted to slots
+  readonly usageTableCols = [
+    msg("Month"),
+    html`
+      ${msg("Running Time")}
```

Comment:
```suggestion
      ${msg("Execution Time")}
```

Since the tooltip makes it clear what this is, might be better to be consistent with the term we're applying for billing?

Reply:
or "Crawl Execution Time"?

Reply:
Going to commit this based on what we decided today